app/views/devise/registrations/new.html.haml

```diff
@@ -21,8 +21,16 @@
       = f.email_field :email, autocomplete: "email", class: 'form-control'
   .row.g-1.g-sm-4
     .col-12.col-sm-10.col-md-8
-      = f.label 'Where did you hear about us?', class: 'col-form-label optional'
-      = f.text_area :heard_about_us, size: "60x4", class: 'form-control'
+      = f.text_field :heard_about_us, class: 'hidden', id: 'heard-about-us'
+      = f.label 'Where did you hear about us?', class: 'col-form-label required'
```

Comment:
Can we add, `for: 'heard-about-us-select'` so that this input is accessible to screen readers?

Typically, we'd just use the attribute name but since we have multiple inputs that we're using for this 1 db value, we have to explicitly set the label's `for` attr

```suggestion
      = f.label 'Where did you hear about us?', class: 'col-form-label required' , for: 'heard-about-us-select'
```